gnovm/cmd/gno/build.go

```diff
@@ -11,13 +11,13 @@ import (
 )
 
 type buildCfg struct {
-	verbose  bool
+	verboseStruct
```

Comment:
I find the previous version more readable. This current version requires exploring multiple files to ensure the expected type.

I suggest considering #731 as it may align with your direction.